examples/demo_dlmbl/solution.py

```diff
@@ -95,62 +145,116 @@
     RandWeightedCropd,
 )
 
-# Trainer class and UNet.
-from viscy.light.engine import VSUNet
-from viscy.light.trainer import VSTrainer
-
+# %%
+# seed random number generators for reproducibility.
 seed_everything(42, workers=True)
 
 # Paths to data and log directory
-data_path = Path(
-    Path("~/data/04_image_translation/HEK_nuclei_membrane_pyramid.zarr/")
-).expanduser()
+top_dir = Path(
+    f"/hpc/mydata/{os.environ['USER']}/data/"
+)  # TODO: Change this to point to your data directory.
+data_path = (
+    top_dir / "06_image_translation/training/a549_hoechst_cellmask_train_val.zarr"
+)
+log_dir = top_dir / "06_image_translation/logs/"
 
-log_dir = Path("~/data/04_image_translation/logs/").expanduser()
+if not data_path.exists():
+    raise FileNotFoundError(
+        f"Data not found at {data_path}. Please check the top_dir and data_path variables."
+    )
 
+# %%
 # Create log directory if needed, and launch tensorboard
 log_dir.mkdir(parents=True, exist_ok=True)
 
 # %% [markdown] tags=[]
 """
-The next cell starts tensorboard within the notebook.
+The next cell starts tensorboard.
 
-<div class="alert alert-danger">
+<div class="alert alert-warning">
 If you launched jupyter lab from ssh terminal, add <code>--host &lt;your-server-name&gt;</code> to the tensorboard command below. <code>&lt;your-server-name&gt;</code> is the address of your compute node that ends in amazonaws.com.
 
-You can also launch tensorboard in an independent tab (instead of in the notebook) by changing the `%` to `!`
+</div>
+
+<div class="alert alert-warning">
+If you are using VSCode and a remote server, you will need to forward the port to view the tensorboard. <br>
+Take note of the port number was assigned in the previous cell.(i.e <code> http://localhost:{port_number_assigned}</code>) <br>
+
+Locate the your VSCode terminal and select the <code>Ports</code> tab <br>
+<ul>
+<li>Add a new port with the <code>port_number_assigned</code>
+<li>Change the port to <code>4000</code> and ensure that the forwarded Adress: <code>localhost:{port_number_assigned}</code>
```

Comment:
What does this mean?

Reply:
If you mean forwarding the random port to 4000 then I don't think it's necessary. It's simpler to have it be the same as what is printed in the cell.